layouts/shortcodes/integrations.html

```diff
@@ -69,70 +79,39 @@
-        {{ else if $v.Params.name }}
-            {{ $.Scratch.Set "formatSlice" (slice)}}
-            {{ range $i, $word := (split $v.Params.name "_") }}
-                {{ $.Scratch.Add "formatSlice" (print (substr $word 0 1 | upper) (substr $word 1 (len $word))) }}
-            {{ end }}
-            {{ $.Scratch.Set "formatname" (delimit ($.Scratch.Get "formatSlice") " ") }}
-        {{ end }}
-        {{ $formatname := $.Scratch.Get "formatname" }}
-
-        {{ $urlname := $v.File.TranslationBaseName }}
-        {{ $src := (print "images/integrations_logos/" ($urlname | lower) ".png")}}
-        {{ $indx := $src }}
-
-        {{/* if  $src */}}
-        {{ if and $indx (fileExists (print "static/" $src)) }}
-            {{ if $formatname }}
-            <div id="mixid_{{$i}}" class="col-6 col-sm-4 col-md-3 col-lg-3 col-xl-2-4 mix text-center {{ range $i, $e := $v.Params.categories }} cat-{{ replace $e "/" "" | urlize }} {{ end }}" data-id="{{ $i }}" data-ref="item">
-                <div id="{{ $v.Params.name | default ($urlname | lower) }}" class="card">
-                    {{ if in $v.Params.categories "marketplace" }}
+    {{ range $integration := $integrationData }}
+        {{ if $integration.int_logo }}
+            <div id="mixid_{{ $integration.id }}" class="col-6 col-sm-4 col-md-3 col-lg-3 col-xl-2-4 mix text-center {{ range $cat := $integration.tags }} {{ $cat }} {{ end }}" data-id="{{ $integration.id }}" data-ref="item">
+                <div id="{{ $integration.name }}" class="card">
+                    {{ if in $integration.tags "cat-marketplace" }}
                         <div class="card-banner">
                             <p class="font-primary font-weight-bold text-uppercase">Marketplace</p>
                         </div>
                     {{ end }}
-                    <a class="card-img" href="{{ (print "integrations/" ($urlname | lower)) | absLangURL }}">
-                        {{ $.Scratch.Set "url" ""}}
-                        {{ if $indx }}
-                            {{ $.Scratch.Set "url" (print (.Site.Params.img_url) $indx "?auto=format") | safeURL }}
-                        {{ else }}
-                            {{ $.Scratch.Set "url" (print (.Site.Params.img_url) "integrations_logos/dd-noimage.png" "?auto=format") | safeURL }}
-                        {{ end }}
-                        {{ $url := $.Scratch.Get "url" }}
+                    <a class="card-img" href="{{ $integration.redirect | absLangURL }}">
                         <picture class="mx-auto">
-                            <source srcset="{{ $url }}&w=144 1x, {{ $url }}&w=144&dpr=2 2x" media="(min-width: 530px)">
-                            <source srcset="{{ $url }}&w=200 1x, {{ $url }}&w=200&dpr=2 2x" media="(min-width: 0px)">
                             <img class="img-fluid mx-auto"
-                                 srcset="{{ $url }}&w=144"
+                                 src="{{ $integration.int_logo }}"
```

Comment:
Just a note for other reviewers; logos are no longer served through imgix so url params are not respected and should be removed